search.js

```diff
@@ -533,38 +533,41 @@ function doSearch() {
 
 			case "google":
 			default:
-				url = 'http://google.com/search?as_rights=(cc_publicdomain|cc_attribute|cc_sharealike' + 
-						((comm) ? "" : "|cc_noncommercial") + ((deriv) ? "" : "|cc_nonderived") + ')' + 
-							rights + '&q=' + query.val(); 
+				url = 'http://google.com/search?as_rights=(cc_publicdomain|cc_attribute|cc_sharealike' +
+					((comm) ? "" : "|cc_noncommercial") + ((deriv) ? "" : "|cc_nonderived") + ')' +
+					rights + '&q=' + query.val();
 				if (lang != null) url += '&hl=' + lang;
 				break;
 
 		}
 
-   	    url = 'https://oldsearch.creativecommons.org/bouncer.php?q=' + query.val() + '&url=' + encodeURIComponent(url);
+		url = 'https://oldsearch.creativecommons.org/bouncer.php?q=' + query.val() + '&url=' + encodeURIComponent(url);
 
-		window.location.href = url;
-//		document.getElementBy$('#stat').setAttribute('src','transparent.gif?engine='+engine+'&comm='+comm+'&deriv='+deriv+'&q='+query.value);
+		window.open(
+			url,
+			'_blank'
```

Comment:
Out of an abundance of caution for security it might be useful to explictly add the `rel=noopener` attribute. 

Example: `window.open(url, '_blank', 'noopener')`



Modern browsers set this implicitly, but we should probably still set it explicitly for any older browser usage. 

## Further Reading
[MDN Window.open()](https://developer.mozilla.org/en-US/docs/Web/API/Window/open)
[About rel=noopener](https://mathiasbynens.github.io/rel-noopener/)